cabal.project

```diff
@@ -44,31 +44,52 @@ package cardano-api
 package cardano-cli
   ghc-options: -Werror
 
+package cardano-client-demo
```

Comment:
If you want to enable this for all packages, I think `package *` should work here:
https://cabal.readthedocs.io/en/3.4/cabal-project.html#package-configuration-options

Reply:
We don't want to enable `Werror` for our dependencies because our dependencies are not controlled by us and may produce warnings when they build.